corehq/apps/sms/forms.py

```diff
@@ -137,10 +137,13 @@ def clean_phone_numbers(self):
                 "number."))
 
         for phone_number in result:
-            validate_phone_number(phone_number)
+            self.validate_phone_number(phone_number)
 
         return result
 
+    def validate_phone_number(self, phone_number: str) -> None:
```

Comment:
can you tell me about this cool new syntax @dannyroberts ?
also why add a method to the object and then call the global method when you can call it directly?

Reply:
The syntax is type hinting, check out discussion here: https://docs.google.com/document/d/1SxDhlyBJVBcn1Jc2GHtIXyDtv2BTBPxiIxoks0lhXGk/

I have the same question about the purpose of the new pass-through method.

Reply:
Actually I think I understand the pass-through: it allows for adding Twilio-specific logic in the subclass `TwilioBackendForm`. This makes sense to me.

Reply:
Yup @orangejenny this is exactly right. @mkangia see ^

Reply:
cool, thanks both.